features/admin.administrators.v1/wizard/steps/admin-user-basic.tsx

```diff
@@ -47,99 +44,140 @@ import {
     useDocumentation
 } from "@wso2is/react-components";
 import { FormValidation } from "@wso2is/validation";
-import { AxiosResponse } from "axios";
 import debounce, { DebouncedFunc } from "lodash-es/debounce";
 import isEmpty from "lodash-es/isEmpty";
 import kebabCase from "lodash-es/kebabCase";
-import React, { FormEvent, ReactElement, useCallback, useEffect, useMemo, useState } from "react";
+import React, {
+    FormEvent,
+    ForwardRefExoticComponent,
+    ForwardedRef,
+    MutableRefObject,
+    ReactElement,
+    RefAttributes,
+    forwardRef,
+    useCallback,
+    useEffect,
+    useImperativeHandle,
+    useMemo,
+    useRef,
+    useState
+} from "react";
 import { useTranslation } from "react-i18next";
 import { useSelector } from "react-redux";
 import { Divider, DropdownProps, Grid, Header } from "semantic-ui-react";
-import { AdministratorConstants } from "../../constants/users";
 import { InternalAdminFormDataInterface } from "../../models/invite";
 import { isAdminUser, isCollaboratorUser } from "../../utils/administrators";
 
 /**
  * Proptypes for the add admin user basic component.
  */
 interface AddAdminUserBasicProps extends IdentifiableComponentInterface {
-    triggerSubmit: boolean;
     onSubmit: (values: any) => void;
     administratorType: string;
     setFinishButtonDisabled: (values: boolean) => void;
+    /**
+     * Roles list for console application.
+     */
+    consoleRolesList: RolesInterface[];
+    /**
+     * Whether the console roles list is loading.
+     */
+    isConsoleRolesListLoading: boolean;
+}
+
+export interface AddAdminUserBasicFormRef {
+    triggerSubmit: () => void;
```

Comment:
Lets document the interface property.

Reply:
Fixed: bec96bb5df36466e776baf4a8f678fdbccadad01